drivers/sensors/paw3222.h

```diff
@@ -0,0 +1,57 @@
+/* Copyright 2024 Colin Lam (Ploopy Corporation)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+#include "pointing_device.h"
+
+#ifndef PAW3222_CS_PIN
+#    ifdef POINTING_DEVICE_CS_PIN
+#        define PAW3222_CS_PIN POINTING_DEVICE_CS_PIN
+#    else
+#        error "No chip select pin defined -- missing POINTING_DEVICE_CS_PIN or PAW3222_CS_PIN define"
+#    endif
+#endif
+
+#ifndef PAW3222_MOTION_PIN
+#    ifdef POINTING_DEVICE_MOTION_PIN
+#        define PAW3222_MOTION_PIN POINTING_DEVICE_MOTION_PIN
+#    else
+#        error "No data pin defined -- missing POINTING_DEVICE_MOTION_PIN or PAW3222_MOTION_PIN"
+#    endif
+#endif
```

Comment:
```suggestion
```

Reply:
Done in [ff65fd0](https://github.com/qmk/qmk_firmware/pull/25763/commits/ff65fd085f10b82f1a17e161c5da6f06e4f0b39b).